config/locales/in_person_proofing/fr.yml

```diff
@@ -92,6 +92,11 @@ fr:
           document d’identité délivré par l’État, votre adresse et votre numéro
           de sécurité sociale.
       state_id:
+        alert_message: Votre carte d’identité délivrée par l’État ne doit pas être
+          périmée. Les pièces d’identité acceptées sont
```

Comment:
We might need to add a colon here after "sont" @JackRyan1989

Reply:
Fixed. 

<img width="495" alt="Screen Shot 2023-02-16 at 11 26 24 AM" src="https://user-images.githubusercontent.com/14644234/219429196-9fc6a9ae-ce42-4304-a9d5-334d3593aa55.png">